pycheribuild/projects/syzkaller.py

```diff
@@ -62,92 +69,135 @@ def setup_config_options(cls, **kwargs):
+        self.targetarch = self.crosscompile_target.cpu_architecture.value
+        if self.targetarch == "aarch64":
+            self.targetarch = "arm64"
+
+        self.get_path()
 
         cheribsd_target = self.get_crosscompile_target(config).get_rootfs_target()
         self.cheribsd_dir = BuildCHERIBSD.get_source_dir(self, cross_target=cheribsd_target)
 
     def syzkaller_install_path(self):
-        return self.config.cheri_sdk_bindir
+        return self.get_sdk_dir(self.config) / "syzkaller"
 
     def syzkaller_binary(self):
-        return self.config.cheri_sdk_bindir / "syz-manager"
+        return self.get_sdk_bindir() / "syz-manager"
 
     def needs_configure(self) -> bool:
         return False
 
+    def _setup_make_args(self):
+        args = {
+            "HOSTARCH": "amd64",
+            "TARGETARCH": self.targetarch,
+            "TARGETOS": "freebsd",
+            "CC": self.CC,
+            "CXX": self.CXX,
+            "PATH": self.get_path()
+        }
+        if self.use_go == GoType.UPSTREAM:
+            args["GOROOT"] = self.goroot.expanduser()
+            args["GOPATH"] = self.gopath.expanduser()
+        self.make_args.set_env(**args)
+
     def compile(self, **kwargs):
         cflags = self.default_compiler_flags + self.default_ldflags
 
-        self.make_args.set_env(
-            HOSTARCH="amd64",
-            TARGETARCH=self.crosscompile_target.cpu_architecture.value,
-            TARGETOS="freebsd",
-            GOROOT=self.goroot.expanduser(),
-            GOPATH=self.gopath.expanduser(),
-            CC=self.CC, CXX=self.CXX,
-            PATH=self._new_path)
+        self._setup_make_args()
+
         if self.sysgen:
             self.generate()
 
         self.make_args.set_env(CFLAGS=" ".join(cflags))
         self.run_make(parallel=False, cwd=self.gosrc)
 
     def generate(self):
-        with self.set_env(PATH=self._new_path, SOURCEDIR=self.cheribsd_dir):
+        with self.set_env(PATH=self.get_path(), SOURCEDIR=self.cheribsd_dir):
             self.run_make("extract", parallel=False, cwd=self.gosrc)
             self.run_make("generate", parallel=False, cwd=self.gosrc)
 
+    def get_every_file_path(self, dir):
+        file_paths = []
+        for path in dir.iterdir():
+            if path.is_dir():
+                sub_files = self.get_every_file_path(path)
+                dir_name = path.parts[-1]
+                for s in sub_files:
+                    s[1] = dir_name + "/" + s[1]
+                    file_paths.append(s)
+            else:
+                file_paths.append([path, path.parts[-1]])
+        return file_paths
+
     def install(self, **kwargs):
         # XXX-AM: should have a propert install dir configuration
         native_build = self.source_dir / "bin"
-        mips64_build = native_build / "freebsd_mips64"
-        syz_remote_install = self.syzkaller_install_path() / "freebsd_mips64"
+        build = native_build
```

Comment:
Either use `build` or `native_build`. I'd pick one and probably also rename it to something more meaningful e.g. `build_output_path`